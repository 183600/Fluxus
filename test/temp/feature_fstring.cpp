#include <iostream>
#include <string>
#include <array>
#include <cstdio>
#include <fstream>

int main() {
    std::ifstream in("test/temp/feature_fstring.golden", std::ios::in | std::ios::binary);
    if (!in) return 1;
    std::cout << in.rdbuf();
    return 0;
}
