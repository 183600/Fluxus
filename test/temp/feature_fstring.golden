Hello, Alice!
Alice is 30 years old.
Alice was born around 1994.
The temperature is 23.5Â°C